operator/internal/status/lokistack.go

```diff
@@ -13,6 +13,12 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// DegradedCondition creates the condition Degraded to the lokistack status conditions.
+type DegradedCondition struct {
```

Comment:
I think `status.Degraded` sounds like a better type name, what do you think?
```suggestion
// Degraded contains information about why the managed LokiStack has an invalid configuration.
type Degraded struct {
```